apps/openchallenges/mcp-server/build.gradle.kts

```diff
@@ -15,6 +14,23 @@ java {
   }
 }
 
+repositories {
+	mavenCentral()
+  mavenLocal()
+}
+
+dependencies {
+	implementation(libs.spring.ai.starter.mcp.server.webflux)
```

Comment:
Spring Web MVC is used when using synchronous endpoints. WebFlux is used when developing async endpoints. This should enable the MCP server to return/stream the response asynchronously bit by bit instead of returning the response all at once, which generates a delay. Overall, the response time is expected to be smaller when using WebFlux.